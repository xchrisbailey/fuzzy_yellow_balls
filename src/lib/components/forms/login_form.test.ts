import { render } from '@testing-library/svelte';
import { describe, expect, it } from 'vitest';
import LoginForm from './login_form.svelte';

describe('login form', () => {
	it('should render a login form', () => {
		const { container } = render(LoginForm);
		expect(container).toMatchSnapshot();
	});
});
