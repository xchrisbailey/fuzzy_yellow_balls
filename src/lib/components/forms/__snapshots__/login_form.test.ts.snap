// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`login form > should render a login form 1`] = `
<body>
  <div>
    <form
      class="flex flex-col gap-4"
      method="POST"
    >
      <label
        class="label"
      >
        <span>
          Email
        </span>
         
        <input
          class="input "
          name="email"
          type="text"
        />
         
      </label>
       
      <label
        class="label"
      >
        <span>
          Password
        </span>
         
        <input
          class="input "
          name="password"
          type="password"
        />
         
      </label>
       
      <button
        class="variant-soft-primary btn"
      >
        Log In
      </button>
    </form>
    <!--&lt;Login_form&gt;-->
  </div>
</body>
`;
